### ISSUE TITLE
optional multiple return values regression

### ISSUE THREAD

**Issue by radare** [vlang/v#4734]
**V version:** V 0.1.26 9e715b8
**OS:** macOS 10.15

**What did you do?**

```go
fn kv(s string) ?(string,string) {
        kw := s.split_nth(':', 2)
        if kw.len != 2 {
                return error('invalid token')
        }
        return kw[0].trim_space(),kw[1].trim_space()
}

fn main() {
	a := 'foo: bar'
	k, v := kv(a) or {
		return
	}
	println('k $k')
	println('v $v')
}
```

**What did you expect to see?**

```
k foo
v bar
```
 
**What did you see instead?**

```
k foo
v
```

**Comment by dkull** [comment]
Taking a look at this

**Comment by HKGx** [comment]
https://github.com/vlang/v/blob/master/vlib/v/gen/cgen.v#L2036
```v
g.expr(node.exprs[0])
``` 
should be changed to
```v
for i, expr in node.exprs {
  g.expr(expr)
  if i < node.exprs.len - 1 {
    g.write(', ')
  }
}
```

**Comment by radare** [comment]
Can you send a PR?

**Comment by dkull** [comment]
That alone won't fix this. Please wait. I am working on this.

**Comment by nedpals** [comment]
Closed. Fixed by #4734.
